get-started/part2.md

```diff
@@ -56,8 +56,11 @@ after doing that, you can expect that the build of your app defined in this
 
 ### `Dockerfile`
 
-Create an empty directory and put this file in it, with the name `Dockerfile`.
-Take note of the comments that explain each statement.
+Create an empty directory. We called ours `getstart/`, but you
```

Comment:
I think the original, more brief explanation was better. Especially because there is actually no reference to `getstart` on this page.